### PR TITLE
Ionosphere

### PR DESCRIPTION
**Pull request by ykempf**

### CODE REVIEW

File: sysboundary/ionosphere.cpp [hunk 1]
```diff
@@ -1313,6 +1314,9 @@ namespace SBC {
 
                  // Local cell
                  std::array<int,3> lfsc = technicalGrid.globalToLocal(fsc[0],fsc[1],fsc[2]);
+                 if(lfsc[0] == -1 && lfsc[1] == -1 && lfsc[2] == -1) {
+                    continue;
+                 }
```
Comment (ursg) [inline sysboundary/ionosphere.cpp:1319]: I would actually suggest that these should be || instead of &&.

**Review by ursg — APPROVED** [review]
Looks pretty good!

File: sysboundary/ionosphere.cpp (hunk 1 above)
Comment (ursg) [reply sysboundary/ionosphere.cpp:1319]: Actually, I'll just fix that myself after merging :)

Comment (ykempf) [reply sysboundary/ionosphere.cpp:1319]: Yeah but fsgrid returns the triplet in any case, not individually. Hence this choice. Checking the first would be enough, as you did in other spots.

**Pull request merged**